docs/make.jl

```diff
@@ -24,6 +24,7 @@ makedocs(modules=[DiffEqPDEBase,DiffEqProblemLibrary],
            "basics/solution.md",
            "basics/plot.md",
            "basics/integrator.md",
+           "basics/structs.md",
```

Comment:
This needs a change. Also it should get a homepage link

Reply:
Oops. Thanks!

Reply:
What do you mean by homepage link?

Reply:
The index of the docs summarizes each section and lists the pages.

Reply:
Like this? https://github.com/JuliaDiffEq/DiffEqDocs.jl/pull/89/commits/8703915d9de64128fdd9909b35e1a31cb2b51e35